contracts/vault/AssetManagement.sol

```diff
@@ -493,6 +493,10 @@ abstract contract AssetManagement is
     //solhint-disable no-empty-blocks
     function _authorizeUpgrade(address) internal override onlyOwner {}
 
+    function isPauseGuardian(address _guardian) external onlyOwner {
```

Comment:
Cannot be only by owner, should be available for everyone

Reply:
Add view word in method signature

Reply:
Alright, done